Wikipedia Citation Templates.js

```diff
@@ -309,6 +316,22 @@ function doExport() {
 				}
 			}
 		}
+
+		// Old publishers may be written as "Last, First" etc. according to
+		// standards, but should be like authors when in citations.
+		// Source: librarian https://it.wikipedia.org/?diff=69781295
+		// "Dozza, Evangelista (1.), eredi"
+		// "Marnef, Jérôme de & Cavellat, Guillaume, veuve"
+		if(item.publisher && properties.date && properties.date < '1831') {
+			// Remove excess space and secondary disambiguations
+			var declutter = item.publisher.replace(/ *\& */g, "; ").replace(/ *\([^)]+\)/g, "");
+			// Shuffle names with disambiguation and remove numbering
+			var shuffle3 = declutter.replace(/ *([^,;]+), +([^,;0-9]+) *[0-9]*, +([^,;]+) */g, "$2 $1 ($3)");
+			// Same, other names
+			var shuffle2 = shuffle3.replace(/ *([^,;]+), +([^,;0-9]+) *[0-9]* */g, "$2 $1");
+			// Use comma list, get rid of space buildup for numbered names
+			properties.publisher = shuffle2.replace(/ *[;] */g, ", ").replace(/  /g, " ");
+		}
```

Comment:
Ok, so I'll move this to Primo.js.

Philipp Zumstein, 01/01/2015 15:28:
> The reordering in these example would be wrong. The publication date
> (<1831) seems for me to be too weak for a condition.

The cases with "&" can easily be fixed by checking that there is a comma 
after it. For the commas, something similar.

We are trying to extract semantics from non-semantic data, there will 
always be errors. The question is at what level this should happen (on 
import, I understand).